application/src/main/java/org/thingsboard/server/controller/EntityRelationController.java

```diff
@@ -129,7 +129,7 @@ public void deleteRelations(@ApiParam(value = ENTITY_ID_PARAM_DESCRIPTION, requi
         checkParameter("entityType", strType);
         EntityId entityId = EntityIdFactory.getByTypeAndId(strType, strId);
         checkEntityId(entityId, Operation.WRITE);
-        tbEntityRelationService.deleteRelations(getTenantId(), getCurrentUser().getCustomerId(), entityId, getCurrentUser());
+        tbEntityRelationService.deleteCommonRelations(getTenantId(), getCurrentUser().getCustomerId(), entityId, getCurrentUser());
```

Comment:
You should rename the method:
```
public void deleteCommonRelations(@ApiParam
```

Reply:
The most important part is missing - you must change the method's description and inform the user that you delete only common relations. Something like:
```
Deletes all the relations ('from' and 'to' direction) for the specified entity and relation type group: 'COMMON'.
```